bundles/org.openhab.binding.renault/src/main/java/org/openhab/binding/renault/internal/api/MyRenaultHttpSession.java

```diff
@@ -87,9 +88,11 @@ private void login() throws RenaultException, InterruptedException, ExecutionExc
         fields.add("ApiKey", this.constants.getGigyaApiKey());
         fields.add("loginID", config.myRenaultUsername);
         fields.add("password", config.myRenaultPassword);
-        logger.debug("URL: {}/accounts.login", this.constants.getGigyaRootUrl());
-        ContentResponse response = httpClient.FORM(this.constants.getGigyaRootUrl() + "/accounts.login", fields);
+        final String url = this.constants.getGigyaRootUrl() + "/accounts.login";
+        ContentResponse response = httpClient.FORM(url, fields);
         if (HttpStatus.OK_200 == response.getStatus()) {
+            logger.debug("GigyaApi Request: {} Response: [{}] {}\n{}", url, response.getStatus(), response.getReason(),
```

Comment:
Dump of full response payload should be **trace** level:
```suggestion
            logger.trace("GigyaApi Request: {} Response: [{}] {}\n{}", url, response.getStatus(), response.getReason(),
```

Reply:
I have changed this.